HunnitLog/HunnitLog/View/Home/ScoreRow.swift

```diff
@@ -0,0 +1,148 @@
+            VStack {
+                HStack {
+                    Circle()
+                        .fill(CustomColor.yellow)
+                        .frame(width: Constants.circleDiameter, height: Constants.circleDiameter)
+                        .padding(.trailing, Constants.trailing)
+                    
+                    VStack(alignment: .leading) {
+                        Text(title)
+                            .font(Constants.titleFont)
+                        Text("달성률 \(achievement)%")
+                            .font(Constants.achievementFont)
+                            .foregroundColor(CustomColor.coolGray)
+                    }
+                }
+                
+                GeometryReader { geometry in
+                    VStack {
+                        HStack {
+                            Emotion(rawValue: makeScore(sliderValue))?
+                                .image
+                                .frame(width: CGFloat(geometry.size.width / 10 * CGFloat(5.9 + sliderValue)), height: 32, alignment: .bottomTrailing)
+                            Spacer()
+                        }
+                    }
+                }
+                .frame(height: 32)
+                Slider(value: $sliderValue, in: -5...5)
+                    .accentColor(CustomColor.lightGray)
+                HStack {
+                    ForEach(Emotion.allCases) { emotion in
+                        Text("\(emotion.rawValue)")
+                        if emotion != .excellent {
+                            Spacer()
+                        }
+                    }
+                }
+            }
+        }
+        .padding(.horizontal, Constants.horizontalPadding)
+        .padding(.vertical, Constants.verticalPadding)
+        .background(Color.white)
+        .makeRoundedWithBorder(CustomColor.gray, width:Constants.borderWidth, cornerRadius: Constants.borderRadius)
+    }
+}
+
+extension ScoreRow {
+    // CaseIterable 은 아래서 Emotion.allCases 를 사용하기 위해 컨펌 해 주었습니다.
+    // Identifiable 은 ForEach 에서 사용하기 위해 컨펌해주었고, 해당 프로토콜을 따르려면 id 변수를 정의해주어야합니다.
+    private enum Emotion: Int, CaseIterable, Identifiable {
+        case bad = -5
+        case notbad = -3
+        case soso = 0
+        case great = 3
+        case excellent = 5
+        
+        var image: Image? {
+            Image(self.imageName)
+        }
+        
+        var id: Int {
+            self.rawValue
+        }
+        
+        private var imageName: String {
+            switch self {
+            case .bad:
+                return "imgTextboxBad"
+            case .notbad:
+                return "imgTextboxNotbad"
+            case .soso:
+                return "imgTextboxSoso"
+            case .great:
+                return "imgTextboxGreat"
+            case .excellent:
+                return "imgTextboxExcellent"
+            }
+            
+        }
+    }
+    
+    
+    // MARK: makeScore - slider의 값을 정해진 점수로 보정해주는 함수입니다.
```

Comment:
요렇게 computed property 로 빼는건 어떨까요?? 
파라미터로 값을 받지 않아도 될 것 같습니다

```suggestion
    private var score: Int {
            let roundedValue = Int(round(sliderValue))
            var score: Int
            if(roundedValue % 2 == 0 && roundedValue != 0) {
                if(roundedValue < 0){
                    score = roundedValue - 1
                } else {
                    score = roundedValue + 1
                }
            } else if(abs(roundedValue) == 1){
                return 0
                
            } else {
                score = roundedValue
                return score
            }
            
            return score
    }
```

Reply:
수정해 두었습니다! computed property를 쓰는 이유가 
`Emotion(rawValue: score)?` 와 같은 형태로 직관적으로 표현하기 위함인가요,
아니면 파라미터로 값을 받지 않기 위함일까요?! 궁금..